docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,581 @@
+
+-  For Arria10 SoC projects:
+
+   -  ``socfpga_arria10-common/zImage``
+   -  ``<target>/fit_spl_fpga.itb``
+   -  ``<target>/socfpga_arria10_socdk_sdmmc.dtb``
+   -  ``<target>/u-boot.img``
+   -  ``<target>/extlinux.conf → extlinux /extlinux.conf``
+   -  ``<target>/u-boot-splx4.sfp`` to the corresponding SD card partition
+      (e.g. ``dd if=preloader_bootloader.bin of=/dev/mmcblk0p3``,
+      usually it is the 3rd partition, which is only a couple of MB in size)
+
+-  For Cyclone5 SoC projects:
+
+   -  ``<target>/soc_system.rbf``
+   -  ``<target>/socfpga.dtb``
+   -  ``<target>/uImage``
+   -  ``<target>/preloader_bootloader.img`` to the corresponding SD card
+      partition (e.g. ``dd if=preloader_bootloader.img of=/dev/mmcblk0p3``)
+
+-  For De10Nano SoC project:
+
+   -  ``socfpga_cyclone5_common/zImage``
+   -  ``<target>/extlinux.conf → extlinux /extlinux.conf``
+   -  ``<target>/u-boot.scr``
+   -  ``<target>/socfpga.dtb``
+   -  ``<target>/soc_system.rbf``
+   -  ``<target>/u-boot-with-spl.sfp`` to the corresponding SD card partition
+      (e.g. ``dd if=u-boot-with-spl.sfp of=/dev/sdd3``)
+
+Examples of building the boot image
+-------------------------------------------------------------------------------
+
+Below is a list of projects supported by us for each carrier to illustrate how
+to build and showcase the different files that are involved in the process. Each
+project has it's own characteristics (some files may be different from a project
+to another).
+
+.. note::
+
+   Pay attention, each project needs it's own specific Linux Kernel Image &
+   Devicetree to be build. Also, follow these instructions to write the file to
+   your SD card, depending on the operating system that you use (Windows or
+   Linux):
+
+   -  :dokuwiki:`[Wiki] Building the Intel SoC-FPGA kernel and devicetrees from source <resources/tools-software/linux-build/generic/socfpga>`
+   -  :dokuwiki:`[Wiki] Linux Download and setting up the image <resources/tools-software/linux-software/zynq_images/linux_hosts>`
+   -  :dokuwiki:`[Wiki] Formatting and Flashing SD Cards using Windows <resources/tools-software/linux-software/zynq_images/windows_hosts>`
+
+Proceed by cloning the kernel tree, point environment to an arm architecture
+cross compiler, build the configuration file, build the image, and lastly build
+the project and board specific device tree.
+
+ADRV9371/Arria10
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  HDL Project: :git-hdl:`here <projects/adrv9371x/a10soc>`
+-  Linux kernel: :git-linux:`here <>`
+
+Building the Linux Kernel image and the Devicetree
+```````````````````````````````````````````````````````````````````````````````
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+
+   user@analog:~$ git clone https://github.com/analogdevicesinc/linux.git
+   user@analog:~$ cd linux/
+   user@analog:~/linux$ export ARCH=arm
+   user@analog:~/linux$ export CROSS_COMPILE=/path/to/your/arm/cross-compiler
```

Comment:
We should give a more specific path.
Maybe [this](https://www.intel.com/content/www/us/en/docs/programmable/683788/19-1/compiling-the-linux-kernel-for-the-intel.html) can help you (there it says CROSS_COMPILE=arm-linux-gnueabihf and I think the tool can find it, but it should be tested).

Reply:
Based on the provided page, I decided to rewrite it like this:
`CROSS_COMPILE=/path_to_cross_compiler/arm-linux-gnueabihf`